tests/unit/LimitedRestorationToFull.spec.ts

```diff
@@ -244,6 +244,7 @@ describe('LimitedRestorationToFull component - edit page', () => {
 
     // enable filing and wait for all queries to complete
     await wrapper.setProps({ appReady: true })
+    wrapper.vm.$data.isDataLoaded = true
```

Comment:
I'm surprised you need to wait for anything to finish after setting `appReady` -- since appReady is the trigger to (async) load a couple of filings.

Reply:
I guess this works, so nothing to change here.